regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -34,6 +36,7 @@ var MainView = Backbone.View.extend({
             this.externalEvents.on('diff:open', this.createView, this);
             this.externalEvents.on('breakaway:open', this.breakawayOpen, this);
             this.externalEvents.on('section:error', this.displayError, this);
+            this.externalEvents.on('section:sethandlers', this.setHandlers, this);
```

Comment:
Should this be outside the `if (Router.hasPushState)` block? I'd think we'd want sticky headers regardless of whether or not the browser supports push state (=> ajax loading)